neutron_policy_server/wsgi.py

```diff
@@ -46,46 +45,129 @@ def fetch_context():
     )
 
 
-# TODO(rlin): Only enable this after neutron bug/2069071 fixed.
-# @app.route("/address-pair", methods=["POST"])
+@app.route("/address-pair", methods=["POST"])
 def enforce_address_pair():
-    # TODO(mnaser): Validate this logic, ideally we should limit this policy
-    #               check only if its a provider network
-    with db_api.CONTEXT_READER.using(g.ctx):
-        network = network_obj.Network.get_object(g.ctx, id=g.target["network_id"])
-    if network["shared"] is False:
```

Comment:
It also surprised my that fetch network shared from DB seems not the way we get `shared` status. will see in followup action to identity the factors from DB for it.